src/main/java/com/redhat/jenkins/plugins/bayesian/Bayesian.java

```diff
@@ -81,7 +81,12 @@ public BayesianStepResponse submitStackForAnalysis(Collection<FilePath> manifest
             try (InputStream in = manifest.read()) {
                 content = ByteStreams.toByteArray(in);
                 builder.addBinaryBody("manifest[]", content, ContentType.DEFAULT_BINARY, manifest.getName());
-                builder.addTextBody("filePath[]", manifest.getRemote(), ContentType.TEXT_PLAIN);
+                String filePath = manifest.getRemote();
+                if (filePath.contains("pom.xml")){
+                    String[] filePathStrs = filePath.split("stackinfo/poms/");
```

Comment:
yeah i know .  as the other ecosystems will not pass through creating effective poms, we have to write a different login for them. i think we can deal with this later